diff -urN aa/videodb/install.php videodb/install.php
--- aa/videodb/install.php	2004-07-16 09:45:21.000000000 +0200
+++ videodb/install.php	2004-09-01 18:21:44.129827168 +0200
@@ -268,7 +268,7 @@
 							'db_password'	=> $db_password,
 							'db_database'	=> $db_database), true);
 
-						if (!file_put_contents($configfile, $config)) {
+						if (!file_put_contents($config_file, $config)) {
 							$error .= '<br/>Could not write config file!
 									   <br/>Please make sure your config file contains the following lines:<br/><br/>'.
 									   highlight_string($config,1);
